lawnchair/src/app/lawnchair/ui/preferences/destinations/HomeScreenPreferences.kt

```diff
@@ -127,6 +127,12 @@ fun HomeScreenPreferences(
                 label = stringResource(id = R.string.show_sys_ui_scrim),
             )
         }
+        PreferenceGroup(heading = stringResource(id = R.string.infinite_scrolling)) {
+            SwitchPreference(
+                prefs.infiniteScrolling.getAdapter(),
+                label = stringResource(id = R.string.infinite_scrolling_label),
+            )
+        }
```

Comment:
1. This should be under the general preferences group, under the double tap preference;
2. Add description to the switch preference for infinite scroll;
3. The strings 
    * Rename `infinite_scrolling_label` to `infinite_scrolling_description` (This will be the description for our switch)
    * Rename `infinite_scrolling` to `infinite_scrolling_label`. (...and this will be the label)